docs/CONNECTOR_PROTOCOL.md

```diff
@@ -19,6 +19,30 @@ Connectors can:
 
 At this stage, our assumption is that one connector will manage one index, and one index will have only one connector associated with it. This may change in the future.
 
+### Data freshness
+
+Data from remote sources are synced via [sync jobs](DEVELOPING.md#syncing), which means there can be data discrepancy between the remote source and Elasticsearch until the next sync job runs.
+
+There are some data sources with streaming option, which allows external systems to capture changes in real-time. This option has been evaluated, and will not in our roadmap in the near future.
+
+If (near) real-time data availability is required, the only option is to [implement incremental sync](DEVELOPING.md#how-an-incremental-sync-works) for the data source, and make it run as frequently a possible.
```

Comment:
This line and the one above read somewhat negatively. Can we spin this to be neutral or positive? Like:

```
In order to maintain (near) real-time data availability, [implement incremental syncing](DEVELOPING.md#how-an-incremental-sync-works) for the data source, with a frequency that matches your needs.
```